airbyte-config/config-persistence/src/test/java/io/airbyte/config/persistence/WorkspacePersistenceTest.java

```diff
@@ -41,6 +41,12 @@ void setup() {
         null));
   }
 
+  @Test
+  void testGetWorkspace() throws ConfigNotFoundException, IOException, JsonValidationException {
+    configRepository.writeStandardWorkspaceNoSecrets(createBaseStandardWorkspace().withWorkspaceId(UUID.randomUUID()));
```

Comment:
So before your bug fix, would this test fail because the returned workspace would have the `UUID.randomUUID()` instead of the `WORKSPACE_ID`? Just making sure I follow the logic of the test

Reply:
exactly.